### ISSUE TITLE
not working on wordpress yet

### ISSUE THREAD

**Issue by karenjuddsmith** [calculoidapp/calculoidapp.github.io#7]
I have downloaded the plugin and input the Calculoid API.
I have created a simple calculator to test how this workis, but it doesn't show up on the wp page.

I have added the shortcode   [calculoid id=1034] on a test page: http://drp.com/test/
---the only calculator I have created thus far but which seems fine on your site.
The page pretty much struggles to load then does not show the calculator.

Any clues?

I am using a responsive theme Divi and this is the test page.
http://drp.com/test/

I have tested to see if any plugins were conflicting, but even when I turn all other plugins off, the calculoid does not appear to work. 

I also turned on the basic 2015 theme (turning off DIVI) to see if the theme was conflicting, but that didn't get the calculator working either.

Any clues?

**Comment by escopecz** [comment]
Here is how I see it:

![drp](https://lh6.googleusercontent.com/Dv9TDhfk4CcA9xwT_KezTBLbbtjsEMa1mTkyLHcKyO5I_DRB6B1e8hmMV-aX-j3DgS01Ywnpw3k=w1831-h849)

It seems to be working for me. Check if you aren't using some old browser. Here is what Calculoid supports:

https://github.com/calculoidapp/calculoidapp.github.io/wiki/Browser-support

**Comment by karenjuddsmith** [comment]
Ok, so I guess it took so long to load I didn't wait around to see it.
I have seen it show up once on the site... but not everytime.

I use the latest versions of google, safari, yosemite and a fast internet,
but the mere presence of the
[calculoid id=1034] seriously slows down the
load of the whole page

And still does NOT load every time (mostly not).
Either in a text area of the page or a widget area.

I would love it to work!

You have done a great job, and would end up putting these on more than one
site (I manage quite a few)
but with this load time.... I just can't and would love to find a way to
use your work.

Is there something that can be done about load time?
The limiting factor seems to be *embed.calculoid.com...*

Karen

**Comment by escopecz** [comment]
I guess you turned off the Calculoid plugin, so I can't test with your web. But I really don't see any lag. I guess it can be caused by your location. embed.calculoid.com server is in Prague, Czech Republic. I tested load from these different locations:

New York: 1.42s
Dallas: 1.36s
Amsterdam: 533s

I guess you experienced a longer load time? Where are you loading it from?

The Calculoid plugin shouldn't slow your whole website down. It just loads the Javascript and CSS. The rest is loaded asynchronously after the page is loaded.

**Comment by karenjuddsmith** [comment]
Yes, I did turn off the plugin as it was messing with the on-page
navigation for this theme.
turning links like this:
/drip-strategies#why/
into

/drip-strategies/#/why

so I just embedded it.

I am currently down south of Miami, using AT&T hotspot, but still not sure
why
they were not showing up...

I currently have the code here:
http://drp.com/drip-strategies/#nofee
but for me, where I am, it is still a now-show and a slow load at that.

I was considering making it a popup, but it would be really nice just to
have it there in the text on the page.

And if this works nicely, there are other cacluators that could be
developed for the site... :)

Karen

**Comment by escopecz** [comment]
We made a several tests from our location which all look good. Could you make 2 tests for us?

1) Go to your website with embedded calculator, press F12. Developer console should show up. Select Network tab and refresh the page. Let it load all the assets. You should see there javascript file loaded from URL embed.calculoid.com. Make sure the embed.calculoid.com is visible. Then make a screen shot. Like this:

![Network tab](https://lh4.googleusercontent.com/apPDK0h_TwECd8ujRZ3ZnKGoRE5uSW_RruWYiEZkK6CdzzqQlSrDBdRmjK52KKvFL9JPrSIzKdY=w1855-h969)

2) Try to run this command:
```
traceroute embed.calculoid.com
```
if you are on unix system (Mac) or this:
```
tracert embed.calculoid.com
```
if you are on Windows. Result should be similar to this:
```
traceroute to embed.calculoid.com (95.168.217.155), 30 hops max, 60 byte packets
 1  myrouter.local (10.0.0.1)  1.146 ms  1.841 ms  2.336 ms
 2  * * *
 3  ip-86-49-52-65.net.upcbroadband.cz (86.49.52.65)  39.485 ms  40.364 ms  40.371 ms
 4  cz-prg02a-ra2-vla2013.net.upc.cz (84.116.222.213)  40.665 ms  40.656 ms  40.298 ms
 5  213.46.172.234 (213.46.172.234)  132.502 ms  132.502 ms  132.100 ms
 6  superhosting.te-5-2-1205.bb1.prg1.cz.m247.com (77.243.181.22)  99.535 ms * *
 7  ttc-c65a-vl6.superhosting.cz (88.86.96.196)  20.175 ms  20.789 ms  21.237 ms
 8  ewww1.easysoftware.cz (95.168.217.155)  20.725 ms  20.750 ms  20.643 ms
```
You can send it to jan@easy.cz

**Comment by karenjuddsmith** [comment]
Hi John:

I will try to do this, but I am on mac and:

1. I can't see the screenshot you sent me. All I get is:
[image: Inline image 1]

Your screenshot image doesn't show up in gmail. I can only see a broken
link that should be to your git hub based screenshot.

Without that image to guide me, I am not sure what I am looking for.

PLUS: The calculoid does not show up most times I try to load it. At it
takes even a simple page more than a minute to stop trying to load and this
was the most recent screenshot.

[image: Inline image 2]
[image: Inline image 3]

I put this screen shot in to show chrome trying to load the page with ONLY
the calculator on it. This lasts for a good 60 seconds then just ends up
with the header and footer and no calculator showing.

That's what I see. Behind the scenes, I am not certain.

This is the code I put in the html text section: (straight from your site)

<link rel="stylesheet" href="https://embed.calculoid.com/styles/main.css" />
<script src="https://embed.calculoid.com/scripts/combined.min.js"></script>
<div ng-app="calculoid" ng-controller="CalculoidMainCtrl"
ng-init="init({calcId:1034,apiKey:'54d4fef15d660'})"
ng-include="load()"></div>


I am not sure if I completed step 1 above as you requested but here is a
screenshot (small cos I am on a laptop)
[image: Inline image 4]

I checked the "errors" and it said there were 2:

[image: Inline image 5]
I am returning from here to San Francisco tomorrow so I am not sureI can
help trouble shoot any more from here. But I will certainly be checking to
see if there are similar issues once I return.

Perhaps we can pick this up from there.

Karen
